spec/unit/provider/debian_spec.rb

```diff
@@ -0,0 +1,136 @@
+#
```

Comment:
Because the new module file exists in `lib/chef/provider/package/<module>.rb`, I think the spec should exist in `spec//unit/provider/package/<module>.rb`.

Reply:
yeah this should definitely get moved.